src/device-state/current-state.ts

```diff
@@ -108,25 +108,25 @@ const getStateDiff = (): DeviceStatus => {
 	}
 
 	const diff = {
-		local: _(stateForReport.local)
-			.omitBy((val, key: keyof DeviceStatus['local']) =>
-				_.isEqual(lastReportedLocal[key], val),
-			)
-			.omit(INTERNAL_STATE_KEYS)
-			.value(),
-		dependent: _(stateForReport.dependent)
-			.omitBy((val, key: keyof DeviceStatus['dependent']) =>
-				_.isEqual(lastReportedDependent[key], val),
-			)
-			.omit(INTERNAL_STATE_KEYS)
```

Comment:
Combining this `omit` into the one above avoids having to loop through the object a second time and create an extra result object which should reduce cpu usage both for the filtering and for the gc